src/game/server/gamemodes/base_pvp/base_pvp.cpp

```diff
@@ -814,7 +816,8 @@ int CGameControllerPvp::OnCharacterDeath(class CCharacter *pVictim, class CPlaye
 				pKiller->GetCharacter()->m_ReloadTimer = 10;
 			}
 		}
-		EndSpree(pVictim->GetPlayer(), pKiller);
+		if(SuicideOrWorld)
+			EndSpree(pVictim->GetPlayer(), pKiller);
```

Comment:
In other modes than fng a death should end the spree.

Reply:
Since this is base controller seems there's no way to check if its fng or not. I can propose to add config variable `sv_end_spree_on_death` that will manage this behaviour

Reply:
There is already ``IsGrenadeGameType()``, ``IsVanillaGameType()`` and ``IsDDRaceGameType()`` could add ``IsFngGameType()`` and then do an if statement in the base controller. You can implement it in gamecontroller.h to return false and overwrite it in base_fng.h to return true.